iosApp/PhongKMMIC/Sources/Application/AppDelegate.swift

```diff
@@ -0,0 +1,20 @@
+//
+//  AppDelegate.swift
+//  PhongKMMIC
+//
+//  Created by Phong Vo on 23/04/2023.
+//  Copyright © 2023 Nimble. All rights reserved.
+//
+
+import UIKit
+
+final class AppDelegate: NSObject, UIApplicationDelegate {
+
+    func application(
+        _ application: UIApplication,
+        didFinishLaunchingWithOptions launchOptions: [UIApplication.LaunchOptionsKey: Any]? = nil
+    ) -> Bool {
+        UITestHelper.shared.speedUpUITestAnimation()
```

Comment:
What do you think about wrapping it with `#if DEBUG`?
It's more safety in the Production 🙏

Reply:
solved in 848e473